tests/Gravatar/Tests/TwigExtensionTest.php

```diff
@@ -8,9 +8,11 @@
 use Twig_Environment;
 use Twig_Loader_Array;
 
-class TwigExtensionTest extends \PHPUnit_Framework_TestCase
+use PHPUnit\Framework\TestCase;
+
+class TwigExtensionTest extends TestCase
 {
-    public function testRegisterExtension()
```

Comment:
Please ether add an assertion or remove the method at all. 🙂

Reply:
Do you mean let this change into ```addRegisterExtension``` method?.

Reply:
I’d say if it is not testing anything than it can be removed. What do you think?

Reply:
Yes, it should be removed because it doesn't do any assertion in this test :+1: .